crates/avalanche-types/src/warp/client.rs

```diff
@@ -0,0 +1,43 @@
+use std::io::{Error, ErrorKind, Read, Result};
+use std::str::FromStr;
+
+use crate::ids::Id;
+use crate::proto::pb::warp::{signer_client, SignRequest, SignResponse};
+use prost::bytes::Bytes;
+use tonic::transport::Channel;
+
+#[derive(Clone)]
+pub struct WarpSignerClient {
+    inner: signer_client::SignerClient<Channel>,
+}
```

Comment:
What's the benefit of the wrapper here? Why not just use the `SignerClient` directly?